docs/rest_api.md

```diff
@@ -240,7 +240,7 @@ default channel, the VLAN drivers choose `vlan/native`.
 
 `GET /networks`
 
-List all networks.
+List all networks or List all public networks.
```

Comment:
Second "List" should not be capitalized.

Also, I would expand this such that it actually describes the conditions under which the call does one or the other (i.e. depending on whether the caller is admin).